projects/ngrx-auto-entity/package.json

```diff
@@ -24,6 +24,7 @@
     "@angular/core": "^8.x",
     "@ngrx/effects": "^8.x",
     "@ngrx/store": "^8.x",
-    "rxjs": "^6.x"
+    "rxjs": "^6.x",
+    "uuidv4": ">=4.x <=6.x"
```

Comment:
@jrista made the changes, though when using `npm install`, I'm still getting peer dependency warnings. In fact, I'm getting them for all the angular stuff too. Any idea why that is? Some cache issue, or did I forget something?

Reply:
I think this is ultimately an artifact of how our build for the library works. I temporarily removed the `postinstall` npm script, and installed, and I was not seeing the peer dependency warnings. I think they only show up when we `npm link` the library. 

That said, I think we should fix the versions of the other packages here. It looks like we are conflating version specifiers: `^8.x` should either be `8.x` or `^8.0.0`. We should fix angular, ngrx and rxjs versions here, then we should be good.